============================= test session starts ==============================
platform linux -- Python 3.6.3, pytest-3.7.0, py-1.5.2, pluggy-0.7.1 -- /home/travis/virtualenv/python3.6.3/bin/python
cachedir: .pytest_cache
rootdir: /home/travis/build/Fondamenti18/fondamenti-di-programmazione/students/1794022/homework01, inifile:
plugins: timeout-1.3.1, json-0.4.0
timeout: 10.0s
timeout method: signal
timeout func_only: False
collecting ... collected 0 items / 1 errors

==================================== ERRORS ====================================
_________________________ ERROR collecting grade02.py __________________________
/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/_pytest/python.py:488: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
<frozen importlib._bootstrap>:971: in _find_and_load
    ???
<frozen importlib._bootstrap>:955: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:656: in _load_unlocked
    ???
<frozen importlib._bootstrap>:626: in _load_backward_compatible
    ???
/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/_pytest/assertion/rewrite.py:226: in load_module
    py.builtin.exec_(co, mod.__dict__)
grade02.py:6: in <module>
    import program02 as program
E     File "/home/travis/build/Fondamenti18/fondamenti-di-programmazione/students/1794022/homework01/program02.py", line 1
E       Python 3.6.2 |Anaconda, Inc.| (default, Sep 20 2017, 13:35:58) [MSC v.1900 32 bit (Intel)] on win32
E                ^
E   SyntaxError: invalid syntax
 generated json report: /home/travis/build/Fondamenti18/fondamenti-di-programmazione/students/1794022/homework01/program02.log.json 
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.23 seconds ============================
